backend/apps/owasp/admin/entity_member.py

```
@@ -42,15 +42,24 @@ class EntityMemberAdmin(admin.ModelAdmin):
             request,
             f"Successfully approved {queryset.update(is_active=True, is_reviewed=True)} members.",
         )
 
     @admin.display(description="Entity", ordering="entity_type")
     def entity(self, obj):
-        """Return entity link."""
+        """
+        Return a clickable admin link to the related entity.
+
+        Example output: a link to the Project/Chapter/Committee admin change page.
+        """
```

Ruff (0.14.5)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)